### ISSUE TITLE
Gradient on netG is computed when updating discriminator

### ISSUE THREAD

**Issue by htzheng** [NVlabs/SPADE#111]
I noticed when updating discriminator netD using `compute_discriminator_loss`, the netG is forwarded with gradient computation. I wonder if one could use `with torch.no_grads()` on netG to reduce computational time?

**Comment by victorca25** [comment]
Don’t you need that to backpropagate the discriminator loss to the generator?

**Comment by htzheng** [comment]
Sorry, that's my mistake...

>
